From 61ff5d4f137ec8b2020537723953b7fc70e2737b Mon Sep 17 00:00:00 2001
From: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
Date: Sat, 19 Sep 2015 13:16:31 +0530
Subject: [PATCH] arm: build u-boot only but run objcopy manually

Due to a bug in CB GCC-5.2 the u-boot.bin is unable to be built,
generate u-boot.bin manually.

Signed-off-by: Srikanth Krishnakar <Srikanth_Krishnakar@mentor.com>
---
 Makefile           | 2 +-
 arch/arm/config.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 9995e00..e9af8e2 100644
--- a/Makefile
+++ b/Makefile
@@ -712,7 +712,7 @@ DO_STATIC_RELA =
 endif
 
 # Always append ALL so that arch config.mk's can add custom ones
-ALL-y += u-boot.srec u-boot.bin System.map binary_size_check
+ALL-y += u-boot.srec u-boot.bin System.map
 
 ALL-$(CONFIG_ONENAND_U_BOOT) += u-boot-onenand.bin
 ifeq ($(CONFIG_SPL_FSL_PBL),y)
diff --git a/arch/arm/config.mk b/arch/arm/config.mk
index 0667984..87b2cf7 100644
--- a/arch/arm/config.mk
+++ b/arch/arm/config.mk
@@ -115,7 +115,7 @@ endif
 ifdef CONFIG_ARM64
 OBJCOPYFLAGS += -j .text -j .rodata -j .data -j .u_boot_list -j .rela.dyn
 else
-OBJCOPYFLAGS += -j .text -j .secure_text -j .rodata -j .hash -j .data -j .got.plt -j .u_boot_list -j .rel.dyn
+#OBJCOPYFLAGS += -j .text -j .secure_text -j .rodata -j .hash -j .data -j .got.plt -j .u_boot_list -j .rel.dyn
 endif
 
 ifdef CONFIG_OF_EMBED
-- 
1.9.1

